templates/taxi/driver_list.html

```diff
@@ -2,6 +2,7 @@
 
 {% block content %}
     <h1>Driver List
+      <a href="{% url 'taxi:driver-create' %}">Create</a>
```

Comment:
Consider adding some styling or placing the 'Create' link in a more prominent location, such as a button with a class like 'btn btn-primary', to make it more noticeable and consistent with other action links/buttons in the application.